resources/views/backend/city/add.blade.php

```diff
@@ -0,0 +1,46 @@
+@extends('backend.layouts.master')
+@section('title', trans('labels.city'))
+@section('content')
+
+<!-- page content --> 
+
+<div class="col-md-12 col-sm-12 col-xs-12">
+	<div class="x_panel">
+		<div class="x_title">
+			<h2>{!! trans('labels.city') !!} <small>{!! trans('labels.create') !!}</small></h2>
+			<div class="clearfix"></div>
+			@if (count($errors) > 0)
```

Comment:
For the validation, I think you should use this package https://github.com/proengsoft/laravel-jsvalidation. 
And for building from, I think you should use Former or Laravelcollective
https://github.com/formers/former or https://laravelcollective.com/docs/5.1/html